src/components/Sidebar/Sidebar.tsx

```diff
@@ -68,7 +68,7 @@ export const Sidebar = ({
       )}
       {data.map(({ label, content }) => (
         <div key={label} className={cn('px-24 mb-32 transition-all', { 'opacity-0': collapsed })}>
-          {label && <SectionTitle className="mb-8">{label}</SectionTitle>}
+          {label && <SectionTitle className="mb-8 !font-bold">{label}</SectionTitle>}
```

Comment:
Do we want this to happen across all `SectionTitle`s? It's also used on the "RightSidebar", seems like both _should_ have the same font weight, but stranger design things happen. If so it would probably be better to make the change [here](https://github.com/ably/docs/blob/main/src/components/Sidebar/SectionTitle.module.css#L2).

Reply:
They did show it on right sidebar in the ticket, I am waiting on designs confirmation still, so we will see.

Reply:
I did do it on on module also, let me see why that did not work.